pool/web/view/component/component_input.ml

```diff
@@ -749,7 +733,14 @@ let multi_select
   in
   div
     ~a:[ a_class (Elements.group_class classnames orientation) ]
-    ([ label [ txt (Elements.input_label language group_field None required) ]
+    ([ label
+         [ txt
+             (Elements.input_label
+                language
```

Comment:
Somewhere, the label of the input element gets the other language then it's actually chosen by the contact (when EN is selected, the label is in German and vice versa).

Reply:
Fixed, refreshing the page when language is updated